pkg/runtime/conversion.go

```diff
@@ -74,6 +74,8 @@ func convertStringSliceToBool(input *[]string, out *bool, s conversion.Scope) er
 	switch strings.ToLower((*input)[0]) {
 	case "true", "1":
 		*out = true
+	case "false", "0":
+		*out = false
 	default:
 		*out = true
```

Comment:
Should probably omit the `case "true", "1"` now - this is confusing at first sight.  Add a comment that only false and 0 resolve to false, all other non-zero length values are true.